docs/src/main/asciidoc/rest-client-reactive.adoc

```diff
@@ -121,14 +121,19 @@ import java.util.Set;
 public interface CountriesService {
 
     @GET
-    @Path("/name/{name}")
-    Set<Country> getByName(@PathParam("name") String name);
+    @Path("/name/{nm}")
+    Set<Country> getByName(String nm);
```

Comment:
Can't we use name here? I really don't see the point in shortening things and making them less clear.

Reply:
The reason I did it was to try to not confuse the reader with `name` everywhere and to make it clear which part needs to match

Reply:
what about `countryName`?

Reply:
Done